examples/convert/eagle3/apply_eagle3_llama4_maverick.sh

```diff
@@ -0,0 +1,6 @@
+speculators convert nvidia/Llama-4-Maverick-17B-128E-Eagle3 \
```

Comment:
Should we make it explicit and set norm_before_residual to False for this checkpoint? even though it defaults to the same thing, I think it'll be nice?